src/Model/Response/Date.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\CdtClient\Model\Response;
+
+/**
+ * Class Date.
+ *
+ * Represents the single date object received from the CDT API.
+ */
+class Date
+{
+    protected string $label;
+
+    /**
+     * @var \DateTimeInterface
+     */
+    protected \DateTimeInterface $date;
+
+    protected string $ecdtDateType;
+
+    protected ?string $tooltip = null;
```

Comment:
Some properties are nullable, unfortunately Swagger does not provide this kind of information.
This is relevant for tests mostly.